src/utils/date-helper.ts

```diff
@@ -1,8 +1,27 @@
 // Converts date to string and pacific date string
 // Example Output: August 11, 2023
+import { isDate } from 'lodash'
+
 export function shortPacificDate (date: Date | string): string {
   return new Intl.DateTimeFormat('en-US', {
     dateStyle: 'long',
     timeZone: 'America/Vancouver'
   }).format(new Date(date))
 }
+
+/**
+ * Converts a Date object to a date string (YYYY-MM-DD) in Pacific timezone.
+ * @example "2021-01-01 07:00:00 GMT" -> "2020-12-31"
+ * @example "2021-01-01 08:00:00 GMT" -> "2021-01-01"
+ */
+export function dateToYyyyMmDd (date: Date): string {
+  // safety check
+  if (!isDate(date) || isNaN(date.getTime())) return null
+
+  return date.toLocaleDateString('en-CA', {
+    timeZone: 'America/Vancouver',
+    month: 'numeric', // 12
+    day: 'numeric', // 31
+    year: 'numeric' // 2020
+  })
+}
```

Comment:
Should use this method from date mixin instead.

Reply:
And actually... instead of using a mixin for date methods, we should have a basic "date utilities" class like here: https://github.com/bcgov/business-filings-ui/blob/main/src/services/date-utilities.ts

It could be under utils/ but I think utils isn't exported, so maybe having it in services/ is a better idea.

Thoughts?

Reply:
I don't believe it needs to be a class, utility functions could be stand-alone. 
I feel it's a bit more portable than a class or mixin but if vue3 converted components are going to remain class-based, than perhaps it would be easier to extend as a class or mixin?

Suppose it boils down to which vue api the project leverages :)

Reply:
I'd like to keep all date methods in some component (class or mixin) that all UIs could use, ie, make it shared.

There's already a shared date mixin, but I'd like to make THIS component not depend on it, so let's just leave alone for now. It's a bit of tech debt unfortunately but it's a lot of churn the other way.

Reply:
Definitely a good point!
Another option we could explore is making a plugin, if the goal was making portable functionality. 
If a plugin was defined and exported with the desired mixins/functionality, it might be barely more than a wrapper per se but there would be benefits on the project side. 

We could include said plugin at the root level of a project, making the functionality available globally without the need to extending or instantiating at the component level.

Anywho, i'm just trailing off down a rabbit hole, this is just theoretical haha.